test/UnitTest/Microsoft.AspNet.OData.Test.Shared/ODataActionParametersTest.cs

```diff
@@ -23,9 +23,9 @@ public class ODataActionParametersTest
 
         [Theory]
         [InlineData("Drive", "Vehicles(Model=6,Name='6')/org.odata.Drive")]
-        [InlineData("Drive", "Vehicles(Model=6,Name='6')/Microsoft.AspNet.OData.Test.Builder.TestModels.Car/org.odata.Drive")]
+        [InlineData("Drive", "Vehicles(Model=6,Name='6')/org.odata.Car/org.odata.Drive")]
```

Comment:
>org.odata [](start = 57, length = 9)

Would prefer to use a namespace other than "org.odata", which sounds official. Maybe something like "My.Schema", or "TestNamespace" or "ns", similar to what is used in some of the other tests?

Reply:
The org.data was not something I introduced. I just changed it to fix the test to actually honor the namespace.